src/modules/Channel/components/Message/index.tsx

```diff
@@ -34,7 +34,7 @@ const Message = (props: MessageProps): React.ReactElement => {
     localMessages,
   } = useChannelContext();
 
-  const { message } = props;
+  const { message, renderRemoveMessageModal = (props) => <RemoveMessageModal {...props} /> } = props;
```

Comment:
wait why did you move this?

Reply:
The purpose is to be able to pass the `renderRemoveMessageModal` function in from a top level parent component that we render within gather app. Prior to this change, the render function was hard coded to use the `RemoveMessageModal` component which is not customizable. This line makes it so that the component is the default but we can override by supplying our own `renderRemoveMessageModal`